src/MigrationTools.Clients.AzureDevops.Rest/Processors/TfsPipelineProcessor.cs

```diff
@@ -5,9 +5,10 @@
 using System.Net;
 using System.Text;
 using Microsoft.Extensions.Logging;
+using MigrationTools.DataContracts.Pipelines;
 using MigrationTools.Endpoints;
 using MigrationTools.Enrichers;
-using MigrationTools.Processors.Pipelines;
+using MigrationTools.Processors;
```

Comment:
This class does not look like the one you deleted in objectmodel? The clien here is still webclient and such

Reply:
I removed the wrong one. Fixed it just now ;-)